Survey/Survey/Resources/LocalizedStrings/en.lproj/Localizable.strings

```diff
@@ -10,4 +10,6 @@
 "password_placeholder" = "Password";
 "login_button" = "Log in";
 "login_failed_title" = "Login failed!";
-"login_failed_description" = "Email or password is wrong, please try again.";
+"login_failed_description" = "Your email or password is incorrect. Please try again.";
+"invalid_email_description" = "Your email is invalid. Please try with new one.";
```

Comment:
What do you think about re-order strings in alphabetical?

Reply:
done in this [commit](https://github.com/Shayokh144/ic-survey-swiftui-combine/pull/44/commits/f8b1f24fbec87c8dd42a17c31afcdba49baa20f7)